syntax = "proto3";

import "error.proto";

message FrameToMarkingDataRequest{
  repeated bytes frame = 1;
  string store_id = 2;
  string inquiry_type = 3; // 문의 종류(주문 문의사항 / 일반 문의사항)
  int32 num = 4; // 주문 번호 or 일반 문의 번호
}

message FrameToMarkingDataResposne{
  bool success = 1; // 성공 여부
  optional EError error = 2; // 에러 코드   
}

service ChangeMiddlware {
  rpc FrameToMarkingData(stream FrameToMarkingDataRequest) returns (FrameToMarkingDataResposne);
}